core/commands/stat_provide.go

```diff
@@ -57,8 +49,8 @@ This interface is not stable and may change from release to release.
 			wtr := tabwriter.NewWriter(w, 1, 2, 1, ' ', 0)
 			defer wtr.Flush()
 
-			fmt.Fprintf(wtr, "TotalReprovides:\t%s\n", humanNumber(s.TotalReprovides))
-			fmt.Fprintf(wtr, "AvgReprovideDuration:\t%s\n", humanDuration(s.AvgReprovideDuration))
+			fmt.Fprintf(wtr, "TotalProvides:\t%s\n", humanNumber(s.TotalReprovides))
+			fmt.Fprintf(wtr, "AvgProvideDuration:\t%s\n", humanDuration(s.AvgReprovideDuration))
```

Comment:
Are the printed names correct? Seems like they should match the variable names, even if that causes a breaking change.

Reply:
We would be making a breaking change – see https://github.com/ipfs/kubo/pull/10779#discussion_r2062684827 

This PR is keeping `stats provide` as-is now and marking it as  deprecated + adds new `stats reprovide` which has new labels.